docs/adding_packages/sources_and_patches.md

```diff
@@ -6,28 +6,195 @@ These are a very important aspects and it helps us to establish the quality of t
+format, the recommendation is to contact the publisher and ask them to provide the sources in a way/format that can be consumed
+programmatically.
 
-* If library sources cannot be downloaded from their official origin or cannot be consumed directly due to their
-  format, the recommendation is to contact the publisher and ask them to provide the sources in a way/format that can be consumed
-  programmatically.
+As a final options, In case of needing those binaries to use them as a "build require" for some library, we will consider following the approach of adding it
+as a system recipe (`<build_require>/system`) and making those binaries available in the CI machines (if the license allows it).
 
-* In case of needing those binaries to use them as a "build require" for some library, we will consider following the approach of adding it
-  as a system recipe (`<build_require>/system`) and making those binaries available in the CI machines (if the license allows it).
+## Supported Versions
+
+In this repository we are building a subset of all the versions for a given library. This set of version changes over time as new versions
+are released and old ones stop being used.
+
+We always welcome latest releases as soon as they are available, and from time to time we remove old versions mainly due to technical reasons:
+the more versions we have, the more resources that are needed in the CI and the more time it takes to build each pull-request (also, the
+more chances of failing because of unexpected errors).
+
+### Removing old versions
+
+The Conan Team may ask you to remove more if they are taking a lot of resources. When removing old versions, please follow these considerations:
+
+* keep one version for every major release
+* for the latest major release, at least three versions should be available (latest three minor versions)
+
+Logic associated to removed revisions, and entries in `config.yml` and `conandata.yml` files should be removed as well. If anyone needs to
+recover them in the future, Git contains the full history and changes can be recovered from it.
+
+Please, note that even if those versions are removed from this repository, **the packages will always be accessible in ConanCenter remote**
+associated to the recipe revision used to build them.
+
+### Adding old versions
+
+We love to hear why in the opening description of the pull requests you need this exact version.
+We usually don't add old versions unless there is a specific request for it.
+
+Take into account that the version might be removed in future pull requests according to the guidelines above.
+Adding versions that are not used by author of the pull request reduces overall resources and time from [the build services](README.md#the-build-service).
+
+## Policy about patching
+
+The main guideline in ConanCenter is to provide already compiled binaries for a set of architectures in the least surprising way as possible, so Conan
```

Comment:
I believe, `least surprising way` needs some strict definition. it's not clear for newcomer what exactly does it mean.